pkg/api/types.go

```diff
@@ -607,6 +607,10 @@ type SecretVolumeSource struct {
 	// the volume setup will error. Paths must be relative and may not contain
 	// the '..' path or start with '..'.
 	Items []KeyToPath `json:"items,omitempty"`
+	// Mode bits to use on created files by default. The used mode bits will
+	// be the provided AND 0777.
+	// Directories within the path are not affected by this setting
```

Comment:
What do you mean with define a default value? A default value for permissions of directories within the path? Right now it's hardcoded and this patch didn't touch it. Do you mean to use a symbolic constant and keep the same default?